### PR TITLE
Pulled the statsd and redis clients out of the API and into the utils repo

### PR DESCRIPTION
**Pull request by minglis**
Straight forward lift and shift, excepting I bumped the version of pytest to match that in the API. Caused 2 warnings. 1 I've fixed, one 1 can't trace.

1st step to sharing them with the Admin application.

### CODE REVIEW

File: tests/test_this_is_setup_properly.py [hunk 1]
```diff
@@ -1,2 +0,0 @@
-def test_for_testing_sake():
-    assert 1 == 1
```
Comment (servingUpAces) [inline tests/test_this_is_setup_properly.py]: what is this?

**Comment by quis** [comment]
This modifies a bunch of the Markdown tests. Bad merge/rebase?

File: tests/test_this_is_setup_properly.py (hunk 1 above)
Comment (minglis) [inline tests/test_this_is_setup_properly.py]: Dunno, I removed it. Think is was the original test from the empty repo to prove pytest worked

**Comment by minglis** [comment]
@quis Yeah. I bumped the version of py.test to match what we use elsewhere and that broke class based testing. So I had to change those tests otherwise they'd be ignored.

**Pull request merged**